toolchest_client/tools/api.py

```diff
@@ -1201,20 +1201,20 @@ def add_database(database_path, tool, database_name, database_primary_name=None,
     :param database_path: Path or list of paths (local or S3) to be passed in as inputs.
     :param tool: Toolchest tool with which you use the database (e.g. toolchest.tools.Kraken2).
     :param database_name: Name of the new database.
-    :param database_primary_name: Name or path of the file to use as the primary database file
-        (i.e., what you would pass into the command line as the database), if uploading multiple
-        files. If unspecified, assumes that the *directory* of files is what will be passed in
-        as the database.
+    :param database_primary_name: Base name of the file/prefix that would normally be passed in
+        to the command line call. Use `database_primary_name=None` to pass the entire *directory*
+        of files as the database.
```

Comment:
I would maybe say something like "to use the directory name as the database". This phrasing to me could read as if the whole directory may not be uploaded if  `database_primary_name=None`isn't used.